### DISCUSSION TITLE
Keyboard shortcut for Next/Previous Keyframe

### DISCUSSION THREAD

**Discussion by andydjames** [keiwando/toonsquid#555] [Ideas]
It would be really helpful to add a keyboard shortcut for Next/Previous Keyframe.
When working in Keyframe mode, one would select the keyframe property by clicking on it so that it is highlighted in the timeline and then have a keyboard shortcut to quickly move to the next or previous keyframe.

It would also be really helpful to ensure that all keyboard shortcuts (if possible) are single keystrokes rather than combined ones (avoid Ctrl + or Shift +)

**Comment by yaknepper** [comment]
I am in absolutely agreement with this suggestion!
